client/src/app/app.ts

```diff
@@ -13,10 +14,13 @@ import { About } from './about/about.component';
 })
 
 @RouteConfig([
-  { path: '/home',       component: Home,        name: 'Home', useAsDefault: true },
-  { path: '/about',      component: About,       name: 'About' },
+  { path: '/home',                component: HomeComponent,          name: 'Home', useAsDefault: true },
```

Comment:
I'm not sure about that, I don't see anyone break the routeing to separate lines, not in Angular style guide, or John Papa one. But I'll check it out before submitting.
And I read now about child routeing, maybe it's better that way, don't know yet..